primitives/src/merkle_tree/prelude.rs

```diff
@@ -64,4 +64,4 @@ impl<F: RescueParameter> DigestAlgorithm<F, F, F> for RescueHash<F> {
 }
 
 /// Example instantiation of a SparseMerkleTree indexed by I
-pub type RescueSparseMerkleTree<I, E, F> = UniversalMerkleTree<E, RescueHash<F>, I, U3, F>;
+pub type RescueSparseMerkleTree<I, F> = UniversalMerkleTree<F, RescueHash<F>, I, U3, F>;
```

Comment:
Any reason for enforcing the element type to be a field element?

Reply:
Actually no. Since we weren't using other types than field element in our code, it just made the initial version of this PR easier.

But I suppose one could ask the same question about [RescueMerkleTree](https://github.com/EspressoSystems/jellyfish/blob/4e017caf3091cee90019911ef330422f808e3f4c/primitives/src/merkle_tree/prelude.rs#L42).

In any case we should probable be consistent.